examples/Pyspark Kernel.ipynb

```diff
@@ -0,0 +1,686 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Features available with the new kernels\n",
+    "\n",
+    "By default Jupyter notebook comes with a `Python2` kernel. A kernel is a program that runs and interprets your code. sparkmagic provides two additional kernels that you can use with Jupyter. These are:\n",
```

Comment:
Jupyter can come with a Python2 kernel, but it can also come with a Python3 kernel